test/unit/sql/select.test.js

```diff
@@ -232,4 +232,16 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
     });
 
   });
+
+  suite('raw query', function () {
+    test('raw replacements', function () {
+      expectsql(sql.selectQuery('User', {
+        attributes: ['*'],
+        having: ['name IN (?)', [1, 'test', 3, "derp"]]
+      }), {
+        default: "SELECT * FROM [User] HAVING name IN (1,'test',3,'derp');",
+        postgres: "SELECT * FROM \"User\" HAVING name IN (ARRAY[1,'test',3,'derp']);"
```

Comment:
No postgres should have the same syntax, or use ANY ARRAY, but generally we use IN for PG aswell.